### PR TITLE
Updated Persian (Farsi) Translation

### PR DESCRIPTION
**Pull request by leopard20**
The original translation was very bad, plus it was missing a lot of stuff.
I didn't bother much with the technical words and the developers stuff.

I also know some French, Spanish and German, so if you want I could update those translations too! (but for now I'd rather let the native speakers step forward)

P.S: Why is PPSSPP inaccessible in Iran? I mean it's given that the Gold Version is not accessible because we don't support any of the Google payment methods. But I don't know what's going on with the free version.
Stupid Google.

### CODE REVIEW

File: fa_IR.ini [hunk 1]
```diff
@@ -1,91 +1,93 @@
 # //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 # Don't Delete This, Please!
+# Current Translation by: Leopard20
+# Original Translator: آراس رضایی
```
Comment (vnctdj) [inline fa_IR.ini]: No need to add these lines, this is already addressed by the "translators**n** =" lines.

File: fa_IR.ini [hunk 2]
```diff
@@ -1,91 +1,93 @@
...
 # لطفا سعی کنید در صورتی که هیچ اطلاعاتی در باره ی این فایل ترجمه ندارید آن را بدون ویرایش بگذارید تا مشکلی در ترجمه به وجود نیاید
 # زیرا ترجمه فارسی این شبیه ساز با مقداری تفاوت ترجمه شده است
 # دوست دار همه ی مردم ایران و صد البته مردم کرد و منطقه ی اورامانات
 # آراس رضایی
 
-#hello henrik, the local name of this lang is "fa_FA" But does not show in the Menu! Please Write The Name Of It "فارسی". Like "العربیه".
+# Corrections and additions by Leopard20
```
Comment (vnctdj) [inline fa_IR.ini]: Same as previous comment.
However feel free to remove the line if needed.

**Comment by leopard20** [comment]
The first two credits lines are necessary IMO. But I removed the "Corrections..." line.

**Comment by hrydgard** [comment]
Hm, I definitely haven't tried to bar Iran. So Google Play works there for free apps?

**Comment by hrydgard** [comment]
Huh, turns out it indeed was turned off in Iran, and also Cuba and Sudan. Maybe they were added but defaulted to off for some reason? From now on, it will be available on Google Play in these countries, should show up in a few hours.

**Comment by hrydgard** [comment]
By the way, if you feel like translating the Play Store description text, feel free to post it here :)

**Pull request merged**